pom.xml

```diff
@@ -709,7 +709,7 @@
             <dependency>
                 <groupId>org.postgresql</groupId>
                 <artifactId>postgresql</artifactId>
-                <version>42.1.4</version>
+                <version>42.2.5</version>
```

Comment:
What changed? Is there any motivation (other than just update)?

Reply:
For example, there were some fixes around SSL and cert validation recently.